### PR TITLE
Parse 'reviewed' event object 'state' property

### PR DESCRIPTION
**Pull request by campesel**
The Timeline Events API can return a `reviewed` event object when listing events for a pull request. One of the properties of [this object](https://docs.github.com/en/developers/webhooks-and-events/events/issue-event-types#reviewed) is `state`, which tells you the state of a submitted review.

The values `state` can have are:

- `commented`
- `changes_requested`
- `approved`

This PR adds this field to the `Timeline` struct defined in `issues_timeline.go` to make it possible to tell the state of a review when listing pull request events.

### CODE REVIEW

**Comment by campesel** [comment]
@googlebot I signed it!

**Comment by campesel** [comment]
Merging this PR would fix issue #1885.

**Review by gmlewis — APPROVED** [review]
Thank you, @campesel !
LGTM.

Awaiting second LGTM before merging.

**Comment by campesel** [comment]
@gmlewis Thanks for the approval!

**Comment by campesel** [comment]
@wesleimp Looks good to you?

**Comment by gmlewis** [comment]
Friendly ping @wesleimp

**Review by Parker77 — APPROVED** [review]
LGTM.

**Pull request merged**
